src/app/view/view-controls/view-controls.component.ts

```diff
@@ -79,15 +80,36 @@ export class ViewControlsComponent implements OnInit, OnDestroy {
   }
 
   public onCopy() {
-    this.router.navigate(['w', this.workspace.organizationCode, this.workspace.projectCode, 'view', this.view.perspective], {
-      queryParams: {
-        query: QueryConverter.toString(this.view.query)
-      }
-    });
+    const path: any[] = ['w', this.workspace.organizationCode, this.workspace.projectCode, 'view', this.view.perspective];
+    this.store.dispatch(new RouterAction.Go({path, query: {query: QueryConverter.toString(this.view.query)}}));
   }
 
   public perspectives(): string[] {
     return Object.values(Perspective);
   }
 
+  public getIconForPerspective(perspective: string): string {
+    switch (perspective) {
+      case Perspective.Table:
+        return 'fa-table';
+      case Perspective.PostIt:
+        return 'fa-file';
```

Comment:
This will be a better fit for smart document perspective. What about using `fa-sticky-note` instead?